apps/backend/src/modules/users/user.controller.ts

```diff
@@ -63,6 +87,45 @@ class UserController extends BaseController {
 		});
 	}
 
+	/**
+	 * @swagger
+	 * /users:
+	 *    delete:
+	 *      tags:
+	 *        - Users
+	 *      security:
+	 *        - bearerAuth: []
+	 *      description: Delete user
+	 *      responses:
+	 *        200:
+	 *          description: Successful operation
+	 *          content:
+	 *            application/json:
+	 *              schema:
+	 *                type: object
+	 *                properties:
+	 *                  success:
+	 *                    type: boolean
+	 */
+	private async delete({
+		params: { id },
+		user,
+	}: APIHandlerOptions<{
+		params: {
+			id: string;
+		};
+		user: UserAuthResponseDto;
+	}>): Promise<APIHandlerResponse> {
+		const userId = Number(id);
+
+		const success = await this.userService.deleteUserByAdmin(user, userId);
```

Comment:
```suggestion
		const success = await this.userService.delete(user, userId);
```
it should be just a delete. you need to handle permissions by the hook